app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -1232,6 +1232,12 @@ public void onAuthorViewClicked() {
         if (media == null || media.getUser() == null) {
             return;
         }
+        if (sessionManager.getUserName() == null) {
+            String userProfileLink = "https://commons.wikimedia.org/wiki/User:" + media.getAuthor();
```

Comment:
I think this base URL (https://commons.wikimedia.org/wiki/) can be replaced by the constant declared in the app, as used in the other fragments too:
https://github.com/commons-app/apps-android-commons/blob/07616485ae213bff99e41b7a507fc32c2fdbc7b1/app/src/main/java/fr/free/nrw/commons/category/CategoryEditHelper.java#L115